app/src/main/java/no/nordicsemi/android/blinky/HiltApplication.kt

```diff
@@ -0,0 +1,16 @@
+package no.nordicsemi.android.blinky
+
+import android.app.Application
+import dagger.hilt.android.HiltAndroidApp
+import timber.log.Timber
+
+@HiltAndroidApp
+class HiltApplication : Application() {
```

Comment:
Why not BlinkyApplication?

Reply:
So that you didn't have to change the name when you build a different app based on Blinky app.
This class does not depend on blinky profile.